app/src/main/java/com/duckduckgo/app/settings/SettingsActivity.kt

```diff
@@ -193,6 +210,7 @@ class SettingsActivity : DuckDuckGoActivity(), SettingsAutomaticallyClearWhatFra
     companion object {
         private const val CLEAR_WHAT_DIALOG_TAG = "CLEAR_WHAT_DIALOG_FRAGMENT"
         private const val CLEAR_WHEN_DIALOG_TAG = "CLEAR_WHEN_DIALOG_FRAGMENT"
+        private const val FEEDBACK_RC = 100
```

Comment:
`RC` doesn't seem clear to me. Can we expand it to `FEEDBACK_RESPONSE_CODE`?

Reply:
Expanded to `FEEDBACK_REQUEST_CODE`. That `RC` was ambiguous between request and response is further evidence I should have done that all along! 😄